src/flashrom/patch/series

```diff
@@ -0,0 +1,3 @@
+0002-Flashrom-support-for-Intel-Rangeley-and-Denverton-CP.patch
+0003-Debian-package-base.patch
+0004-Flashrom-remove-tmpl.patch
```

Comment:
What is the purpose of the patch 0004?

Reply:
0002 patch will add target "flashrom.8.tmpl" on dpkg-buildimage call.
But flashrom github repo didnt have any rules for "flashrom.8.tmpl". 
Hence,  dpkg-buildpackage command will get fail.
To avoid this 0004 patch is applied to remove "flashrom.8.tmpl" related contents in Makefile.